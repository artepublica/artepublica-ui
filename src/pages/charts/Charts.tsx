import { StyleSheet } from 'react-native';
import { SafeAreaView } from 'react-native-safe-area-context';

import { Text } from '@components';

function Charts(): JSX.Element {
    return (
        <SafeAreaView style={styles.container}>
            <Text>Charts</Text>
        </SafeAreaView>
    );
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        alignItems: 'center',
        justifyContent: 'center',
    },
});

export default Charts;
