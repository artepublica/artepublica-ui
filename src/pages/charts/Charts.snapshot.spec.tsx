import { create } from 'react-test-renderer';

import Charts from './Charts';

describe('<Charts />', () => {
    it('default snapshot', () => {
        const json = create(<Charts />).toJSON();

        expect(json).toMatchSnapshot();
    });
});
