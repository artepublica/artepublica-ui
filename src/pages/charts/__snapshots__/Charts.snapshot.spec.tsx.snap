// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Charts /> default snapshot 1`] = `
<RNCSafeAreaView
  edges={
    Array [
      "bottom",
      "left",
      "right",
      "top",
    ]
  }
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "fontFamily",
        },
        undefined,
      ]
    }
  >
    Charts
  </Text>
</RNCSafeAreaView>
`;
